resources/assets/coffee/react/beatmap-discussions/mode-switcher.coffee

```diff
@@ -16,7 +16,7 @@ export class ModeSwitcher extends React.PureComponent
     @scrollerRef = React.createRef()
 
 
-  componentDidUpdate: =>
+  componentDidMount: =>
```

Comment:
`componentDidUpdate` still needs to call `scrollTo` to bring the clicked item completely into view when it's only partially visible

https://user-images.githubusercontent.com/1694544/121883720-0da49d80-cd4d-11eb-877e-9a82a05c5452.mov

Reply:
Yep, just notice about that.